### PR TITLE
Update to the latest 8.x-1.x

### PR DESCRIPTION
**Pull request by m4olivei**
This updates to the latest 8.x-1.x. Many of the patches we were using have been incorporated upstream. We're almost at parity except for two patches. I'll see if I have some time to push that along.

Here is the procedure I went through to update:

```
git checkout 8.x-1.x
git pull lullabot 8.x-1.x # lullabot is a remote for git@github.com:Lullabot/media_mpx.git
git checkout -b media-mpx-update-8.x-1.x
git merge patched
<resolve conflicts>
git add src/Plugin/Field/FieldFormatter/PlayerFormatter.php
git add src/Access/MediaAvailableAccess.php
git commit
```

Sanity checks:

```
git diff --stat 8.x-1.x # Shows only the remaining patches over and above 8.x-1.x
```

### CODE REVIEW

File: PATCHES.md [hunk 1]
```diff
@@ -1,12 +1,5 @@
-Issue #91 \Drupal\media_mpx\Plugin\Field\FieldFormatter\PlayerFormatter makes extending hard redux
- * https://patch-diff.githubusercontent.com/raw/Lullabot/media_mpx/pull/92.patch
-BR-6856: Hack date conversion to map to unix timestamps instead of datetimes
- * See `23316172e425041af31ff9f378f3cfa54c1b843e`
 Issue #108: Use mapped dates when calculating media availability
```
Comment (mattwith) [inline PATCHES.md]: It looks like we should remove this line and the line below it.

**Comment by salvamomo** [comment]
\o hey there. Just a heads up that we pushed a few more changes to media_mpx 8.x-1.x, in case you want to incorporate those in this PR too 👍

**Comment by m4olivei** [comment]
@salvamomo orly!? Nice. Sounds like it removes the need for one of our patches too. Thanks for pointing that out. Will definitely pull it in.

**Comment by m4olivei** [comment]
Recent updates hot off the press merged in. `PATCHES.md` file updated.

**Comment by m4olivei** [comment]
Noticed a bad merge conflict fix when I compared the diff. Now the diff is nice and clean, showing only the two file changed that are open upstream pull requests. Correctly reflected in the `PATCHES.md` file. 💅

**Pull request merged**